packages/ui-common/package.json

```diff
@@ -3,9 +3,10 @@
     "version": "1.2.4",
     "type": "module",
     "scripts": {
-        "build": "yarn clean && yarn generate && tsc --project tsconfig.build.json && fix-esm-import-path dist > /dev/null",
+        "build": "yarn clean && yarn generate && yarn run -T tsc --project tsconfig.build.json && fix-esm-import-path dist > /dev/null",
```

Comment:
Long options in scripts please. `yarn run --top-level`. I had to go digging through the docs to even figure out what `-T` means as it's not common.